transport_nantes/mobilito/static/mobilito/js/geolocation_map.js

```diff
@@ -99,12 +99,16 @@ function onMapClick(e) {
     isLoading = true;
     // Geocode the latitude and longitude of the click and display the address.
     getGeocodedAddress(e.latlng.lat, e.latlng.lng).then(function (data) {
-        let addr = data.results[0].formatted_address;
-        marker = L.marker(e.latlng).addTo(map)
-            .bindPopup(`${addr}`).openPopup();
-        $('#id_location').val(`${addr}`)
-        $('#id_latitude').val(`${e.latlng.lat}`)
-        $('#id_longitude').val(`${e.latlng.lng}`)
+        if (data.status === "OK") {
+            let addr = data.results[0].formatted_address;
+            marker = L.marker(e.latlng).addTo(map)
+                .bindPopup(`${addr}`).openPopup();
+            $('#id_location').val(`${addr}`)
+            $('#id_latitude').val(`${e.latlng.lat}`)
+            $('#id_longitude').val(`${e.latlng.lng}`)
+        } else {
+            alert("La localisation n'est pas disponible pour le moment.")
```

Comment:
This needs a pointer to documentation so that any future dev can understand that the  specification of this service says that "data.status will be 'OK' on success, and anything else on failure with no further nuance of meaning".

That would be a rather rubbish specification for an service, however, so a future dev would be right in wondering if the error was the dev who wrote our code, and so go off searching.  You need to cut that search short by explaining here that the foreign service really provides such a poor interface and providing a pointer to it.

Reply:
Added link to maps' docs.

They actually have other status codes, but the ultimate result on the user's end is that they don't get a formatted address.